scylla/Cargo.toml

```diff
@@ -77,11 +78,13 @@ base64 = { version = "0.22.1", optional = true }
 rand_pcg = "0.3.1"
 socket2 = { version = "0.5.3", features = ["all"] }
 lazy_static = "1"
+tabled =  { version = "0.17.0", features = ["std", "ansi"], optional = true }
+inline_colorization = { version = "0.1.6", optional = true }
+num-bigint-04 = { package = "num-bigint", version = "0.4" }
+bigdecimal-04 = { package = "bigdecimal", version = "0.4" }
```

Comment:
These should be made `optional = true` here and left as mandatory in dev-dependencies.